admin/templates/management/commands.html

```diff
@@ -0,0 +1,22 @@
+{% extends 'base.html' %}
+
+{% block content %}
+<h1> Management Commands </h1>
```

Comment:
nit: If you're possibly going to add more commands to this page you should try to make this page a table or have it appear more list-like. The "Run" button for example is under the main text, it will be hard to read when other commands are added.